build/localmode/RunSamples.cmd

```diff
@@ -85,12 +91,17 @@ pushd "%SPARKCLR_HOME%\scripts"
 @cd
 @dir /s "%SPARKCLR_HOME%"
 
-if "!USER_EXE!"=="" (
-    @echo [RunSamples.cmd] call sparkclr-submit.cmd --jars %SPARKCLR_EXT_JARS% -exe SparkCLRSamples.exe %SAMPLES_DIR% spark.local.dir %TEMP_DIR% sparkclr.sampledata.loc %SPARKCLR_HOME%\data %*
-    call sparkclr-submit.cmd --jars %SPARKCLR_EXT_JARS% --exe SparkCLRSamples.exe %SAMPLES_DIR% spark.local.dir %TEMP_DIR% sparkclr.sampledata.loc %SPARKCLR_HOME%\data %*
+if !INTERACTIVE! == "interactive" (
+	@echo [RunSamples.cmd] call sparkclr-repl.cmd %*
+	call sparkclr-repl.cmd
```

Comment:
+1 Do we really need to add an interactive option to RunSamples, given that we already have sparkclr-repl.cmd under scripts.?